beeai/agents/backport_agent.py

```diff
@@ -7,6 +7,7 @@
 
 import aiohttp
 from pydantic import BaseModel, Field
+from agents.models import Task, BackportInputSchema as InputSchema, BackportOutputSchema as OutputSchema
```

Comment:
Just wondering if it wouldn't make sense to move the model definitions one level higher, to already existing [common.models](https://github.com/packit/ai-workflows/blob/67073b91373228cfc19805512f7c12e1e17f55d7/beeai/common/models.py), so we have them all in one place?